config/default.yml

```diff
@@ -1452,6 +1452,16 @@ Lint/InheritException:
     - runtime_error
     - standard_error
 
+Lint/MissingCopEnableDirective:
+  # Maximum number of consecutive lines the cop can be disabled for.
+  # 0 allows only single-line disables
+  # 1 would mean the following is the maximum allowed is:
+  #   # rubocop:disable SomeCop
+  #   a = 1
+  #   # rubocop:enable SomeCop
+  # .inf for any size
+  MaximumRangeSize: .inf
```

Comment:
I'm not sure I like this name, as it's a bit misleading IMO.

I also wonder why for the unbounded case we don't use nil, as we normally do.

Reply:
This is how `Infinity` is encoded in YAML.

http://www.yaml.org/refcard.html

Reply:
Got it. Thanks!